guardian/workflow-frontend
# Splits preview link from open in column links

The "Open In" column icons are seldom used except preview, which is generally not known about. The icons are taking up valuable screen space when they aren't needed. This PR splits out the "Preview" icon into a separate column. [Trello card](https://trello.com/c/VWMAO4Rb/1879-split-preview-link-from-openin-column)

![image](https://user-images.githubusercontent.com/4633246/82215942-0e7fbb80-9910-11ea-9299-56ee3ed99215.png)

The advantages of this are better use of available screen space and an additional column heading which labels the icon more clearly. 

I have also increased the size of the preview icon for good measure. 



### How to review and test

Open the Workflow dashboard, enable the "Preview" column, observe the correct link and display behaviour.
